ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -1252,6 +1252,21 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-lifecycle-advanced
+- as: aws-rosa-hcp-upgrade
```

Comment:
I think we need a `fN (N=1,2,3,...)` in the name to indicate it's test frequency.

Reply:
@liangxia This job isn't triggered periodically. We will trigger it manually. So I set the cron "0 2 15 4 *". Can we not set the fN for this scenario? Or which value do you recommend?

Reply:
For jobs that will triggered manually or api, we use `f999`.

Reply:
Thanks.I will update it now